### PR TITLE
remove '.' from the end of urls.

### PR DESCRIPTION
**Pull request by brandondutra**
ipython links the url with the '.' when printing things like "Please click http://google.com."

### CODE REVIEW

File: google/datalab/contrib/mlworkbench/commands/_ml.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def ml(line, cell=None):
               num_workers: Dataflow number of workers. If not set, DataFlow
               service will determine the number.
               worker_machine_type: a machine name from
-                  https://cloud.google.com/compute/docs/machine-types. If not
+                  https://cloud.google.com/compute/docs/machine-types If not
```
Comment (jimmc) [inline google/datalab/contrib/mlworkbench/commands/_ml.py]: Maybe move "If not" to the start of the next line.

File: google/datalab/contrib/mlworkbench/commands/_ml.py [hunk 2]
```diff
@@ -206,7 +206,7 @@ def ml(line, cell=None):
               num_workers: Dataflow number of workers. If not set, DataFlow
               service will determine the number.
```
Comment (jimmc) [inline google/datalab/contrib/mlworkbench/commands/_ml.py]: I don't know how textwrap.dedent is supposed to work, but should this line be indented one tab stop, such as line 209?

Comment (brandondutra) [inline google/datalab/contrib/mlworkbench/commands/_ml.py]: thanks!

File: google/datalab/contrib/mlworkbench/commands/_ml.py (hunk 1 above)
Comment (brandondutra) [inline google/datalab/contrib/mlworkbench/commands/_ml.py]: done

**Pull request merged**